drivers/i2c/i2c_ll_stm32.c

```diff
@@ -43,26 +43,39 @@ LOG_MODULE_REGISTER(i2c_ll_stm32);
 #else
 #define STM32_I2C_DOMAIN_CLOCK_SUPPORT 0
 #endif
+int i2c_stm32_get_config(const struct device *dev, uint32_t *config)
```

Comment:
Nipick: please break a line before this function.

Reply:
done